src/reducers/index.js

```diff
@@ -39,7 +38,7 @@ import workflow from './workflow/workflow';
  * @param {Object} action Action to be handled.
  * @returns {Object} New state.
  */
-export default combineReducers({
+const ploneReducers = {
```

Comment:
Better just call this reducers and not prefix with plone

Reply:
ok

Reply:
@robgietema done.